website/content/docs/ecs/configuration-reference.mdx

```diff
@@ -44,9 +44,110 @@ These are the top-level fields for the Consul ECS configuration format.
 | Field | Type | Required | Description |
 | ----- | ---- | -------- | ----------- |
 | `bootstrapDir` | `string` | required | The directory at which to mount the shared volume where Envoy bootstrap configuration is written by `consul-ecs mesh-init`.  |
+| `consulCACertFile` | `string` | optional | The file path of the Consul server CA certificate.  |
+| `consulHTTPAddr` | `string` | optional | The HTTP(S) URL of the Consul server. Required when `authMethod.enabled` is set  |
+| [`consulLogin`](#consullogin) | `object` | optional | Configuration for login to the AWS IAM auth method.  |
+| [`gateway`](#gateway) | `object` | optional | Configuration for the gateway proxy registration.  |
 | `healthSyncContainers` | `array` | optional | The names of containers that will have health check status synced from ECS into Consul. Cannot be specified with `service.checks`.  |
+| `logLevel` | `string` | optional | Sets the log level for the `consul-ecs mesh-init` and `consul-ecs health-sync` commands. Defaults to `INFO`. Must be one of `TRACE`, `DEBUG`, `INFO`, `WARN`, `ERROR`, or `null`. |
 | [`proxy`](#proxy) | `object` | optional | Configuration for the sidecar proxy registration with Consul.  |
-| [`service`](#service) | `object` | required | Configuration for Consul service registration.  |
+| [`service`](#service) | `object` | optional | Configuration for Consul service registration.  |
+
+# `consulLogin`
+
+Configuration for login to the AWS IAM auth method.
+
+| Field | Type | Required | Description |
+| ----- | ---- | -------- | ----------- |
+| `enabled` | `boolean` | optional | Enable login to Consul's AWS IAM auth method to obtain an ACL token. This requires the auth method to be configured on the Consul server, and the ECS task role must be trusted by the auth method. After login, the token is written to the file `<bootstrapDir>/service-token`.  |
+| `extraLoginFlags` | `array` | optional | Additional CLI flags to pass to the `consul login` command. These are appended to the command `consul login -type aws -method <name> -token-sink-file <file> -aws-auto-bearer-token -aws-include-identity`.  |
+| `includeEntity` | `boolean` | optional | Defaults to true. Set this to false to remove the `-aws-include-entity` flag from the `consul login` command. The `-aws-include-entity` flag should be passed if and only if the Consul AWS IAM auth method is configured with `EnableIAMEntityDetails=true`.  |
+| `method` | `string` | optional | The name of Consul auth method. This is passed as the `-method` option to the `consul login` command. Defaults to `iam-ecs-service-token`.  |
+
+# `gateway`
```

Comment:
Note: This page is auto-generated documentation. It reflects latest in the hashicorp/consul-ecs repo, however the gateway work is still in progress, so the gateway reference config here will be revised prior to the 0.5 release when we want to make these docs go live.